packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -15,6 +15,25 @@ export interface GraphQLClient {
    */
   request<T>(query: string | DocumentNode, variables?: { [key: string]: unknown }): Promise<T>;
 }
+/**
+ * Defines the strategy for retrying GraphQL requests based on errors and attempts.
+ */
+export interface RetryStrategy {
```

Comment:
Please make sure RetryStrategy type is exported by sitecore-jss-nextjs, react, angular, vue packages and can be used by sample apps.

Reply:
Good point